config.js

```diff
@@ -30,9 +30,12 @@ function scanConfigObject(configObject) {
   let filteredKeys = Object.keys(configObject).filter((key) => {
     return !["date", "version"].includes(key);
   });
-  if (configObject.REDIS_AUTH_ENABLED.toLowerCase() === "false") {
```

Comment:
Let's change the logic to **!== "true"**, same for the other place. So that the only way to enable Redis Auth is set it to **true** any other value will be treated as **false**. Sorry for the back and forth!